.travis.yml

```diff
@@ -0,0 +1,12 @@
+dist: trusty
+sudo: false
+language: go
+
+go:
+  - "1.10"
+
+jobs:
+  include:
+    - stage: test
+      script: bash tests/setup.sh && bash tests/run.sh
+    - script: bash tests/setup.sh && bash tests/lint.sh
```

Comment:
Minor, but still: probably it would be better to use shebangs and just run `tests/setup.sh` instead of `bash tests/setup.sh`. Also, I don't see a reason why it should actually use `bash` and not `sh`.